app/code/community/Sailthru/Email/Model/Client/Content.php

```diff
@@ -64,7 +65,7 @@ public function getProductData(Mage_Catalog_Model_Product $product)
 
         $isSimple = ($productType == 'simple');
         $parents = Mage::getModel('catalog/product_type_configurable')->getParentIdsByChild($product->getId());
-        $isVariant = ($isSimple and (sizeof($parents) == 1));
+        $isVariant = ($isSimple and (sizeof($parents) > 0)); // Could have more then 1 parent.
```

Comment:
True that a product can multiple parents. But a variant (e.g. small blue shirt where shirt is a configurable product) would only have one parent, no?

What product use-cases have multiple parents?